### ISSUE TITLE
Latest version used on Ubuntu 14.04 LTS can't find nettle

### ISSUE THREAD

**Issue by bl4** [rdp/ffmpeg-windows-build-helpers#111]
But it looks like nettle is there.  Please advise.

> wget https://raw.github.com/rdp/ffmpeg-windows-build-helpers/master/cross_compile_ffmpeg.sh -O cross_compile_ffmpeg.sh
> chmod u+x cross_compile_ffmpeg.sh
> 
> ./cross_compile_ffmpeg.sh
> 
> .
> .
> .
> checking for a sed that does not truncate output... /bin/sed
> checking for autogen... autogen
> checking for inline... inline
> checking for ANSI C header files... (cached) yes
> checking cpuid.h usability... yes
> checking cpuid.h presence... yes
> checking for cpuid.h... yes
> checking for getrandom... no
> checking for getentropy... no
> checking for NETTLE... no
> configure: error: 
>   ***
>   *** Libnettle 3.1 was not found.
> 
> kuuser@kubuntu-VirtualBox:/mnt/oss/rdp_ffmpeg$ ls 
> cross_compile_ffmpeg.sh  sandbox/                 
> kuuser@kubuntu-VirtualBox:/mnt/oss/rdp_ffmpeg$ ls sandbox/
> cross_compilers/ x86_64/          
> kuuser@kubuntu-VirtualBox:/mnt/oss/rdp_ffmpeg$ ls sandbox/x86_64/nettle-3.1/
> aclocal.m4                                                  gcm-aes.o
> aes128-meta.c                                               gcm-aes.o.d
> aes128-meta.o                                               gcm.c
> aes128-meta.o.d                                             gcm-camellia128.c
> aes128-set-decrypt-key.c                                    gcm-camellia128-meta.c
> aes128-set-decrypt-key.o                                    gcm-camellia128-meta.o
> aes128-set-decrypt-key.o.d                                  gcm-camellia128-meta.o.d
> aes128-set-encrypt-key.c                                    gcm-camellia128.o
> aes128-set-encrypt-key.o                                    gcm-camellia128.o.d
> aes128-set-encrypt-key.o.d                                  gcm-camellia256.c
> aes192-meta.c                                               gcm-camellia256-meta.c

**Comment by rdp** [comment]
could you try it again? somebody else said it was working...

**Comment by bl4** [comment]
Yes, it looks like it's working.  I'll report back.

**Comment by bl4** [comment]
truncated:
https://gist.github.com/bl4/0a0576f64851fa9f3c63db0a9bed1c41

full:
https://gist.github.com/bl4/0a0576f64851fa9f3c63db0a9bed1c41/raw/bacfa0f39c4e3b9761a00177e9bf3763940a247d/gistfile1.txt

**Comment by rdp** [comment]
yeah try the latest again, that vector-gen issue should be fixed now, sorry
for the noise
